kgforge/core/forge.py

```diff
@@ -224,13 +224,47 @@ def validate(self, data: Union[Resource, List[Resource]],
 
     # Resolving User Interface.
 
-    def resolvers(self):
-        print("Available scopes:")
-        for k, v in sorted(self._resolvers.items()):
-            print(" - ", k, ":")
-            for r_key, r_value in v.items():
-                print("     - resolver: ", r_key)
-                print("         - targets: ",  ",".join(r_value.targets.keys()))
+    def resolvers(self, as_dict: bool = False) -> Optional[Dict]:
```

Comment:
forge.template() uses `output: str` to control the type of output. I recommend to reuse 'output' instead of 'as_dict'

Reply:
So in that case `output` should be by default `True` in order to keep backwards compatibility with current implementation and when it is False the function should return the dictionary instead of printing.

Reply:
I just noticed that in your comment you mention `output` to be `string`. How do you think of it being used?

Reply:
It is used as in: https://github.com/BlueBrain/nexus-forge/blob/b60fe2163db5fa331e7edb3bf2e5f86381c5cf0a/kgforge/core/archetypes/model.py#L93

Reply:
I modified the implementation to accept string instead of bool and pushed the changes